test/Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM debian:bullseye
 
 # Required for building
-RUN apt-get update && apt-get install -y \
+RUN apt-get update -q && apt-get install -q -y --no-install-recommends \
+  gcc \
```

Comment:
These need to be added since it was installed as a recommended package before?
Maybe give a short hint in the commit message.